modules/account-lib/src/index.ts

```diff
@@ -8,8 +8,10 @@ import {
   Ed25519BIP32,
   Ed25519KeyDeriver,
   Eddsa,
+  Ecdsa,
+  BIP32,
 } from '@bitgo/sdk-core';
-export { Eddsa, Ed25519BIP32 };
+export { Eddsa, Ed25519BIP32, Ecdsa, BIP32 };
```

Comment:
Ecdsa, BIP32 is available from sdk core, Can you directly use it from sdk-core package, I don't think we need to export here also

Reply:
Only exporting BIP32 now